repository/Seaside-Tests-Core.package/WAComboResponseTest.class/instance/response.st

```diff
@@ -1,3 +1,7 @@
 accessing
 response
-	^ super response ifNil: [ response := WAComboResponse external: (GRPlatform current writeCharacterStreamOn: (String new: 4096)) ]
\ No newline at end of file
+	^ super response ifNil: [
+		response := WAComboResponse
+			onStream: (GRPlatform current writeCharacterStreamOn: (ByteArray new: 1024))
```

Comment:
This doesn't look right. `#writeCharacterStreamOn:` wants a `String`, should we instead send `#writeCharacterStreamOn:`?

Reply:
It should be something else, yes. I presume, you meant to write `writeBinaryStreamOn:`?

Reply:
> I presume, you meant to write writeBinaryStreamOn:?

Yes

> I remember having to fight a lot with the wrapping streams (i.e., Zinc).

Let's figure out the problem and solve it.

Reply:
`#writeCharacterStreamOn:` simply delegates to `WriteStream on:`, which operates on collections, so in that sense it's fine. Also, this is in a test and there is no `#writeBinaryStreamOn:` so far. I'll just leave it if you don't object.